bundle/src/main/java/com/adobe/acs/commons/workflow/bulk/BulkWorkflowEngine.java

```diff
@@ -35,6 +35,8 @@
 
     String SLING_FOLDER = "sling:Folder";
 
+    String BULK_WORKFLOW_MANAGER_PAGE_FOLDER_PATH = "/etc/acs-commons/bulk-workflow-manager";
```

Comment:
Not a big deal, but does this constant need to be in the API? Would allow avoiding the export version bump.